CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Contributing Guidelines
+
+🎉 First off: Thank you for your interest in contributing to our project 🥳
+
+Whether it's a bug report, new feature, correction, or additional documentation, we greatly value feedback and contributions from our community.
+
+Please read through this document before submitting any issues or pull requests. This ensures we have all necessary information to respond to your bug report or contribution.
+
+
+## ❤️ Code of Conduct 
+
+This project has adopted the [Contributor Covenant Code of Conduct](CODE_OF_CONDUCT.md). Before contributing, please take the time to read our COC. Everyone participating is expected to uphold this code. Please report unacceptable behavior to us!
+
+For more information see the [Code of Conduct FAQ](https://www.contributor-covenant.org/faq/).
+
+### Guideline commit messages
```

Comment:
I'd convert this to level 2 title, I think it doesn't belong under "Code of Conduct"
```suggestion
## Guideline commit messages
```